association-backend/src/association/auth.py

```diff
@@ -0,0 +1,18 @@
+from pythonit_toolkit.starlette_backend.pastaporto_backend import PastaportoAuthBackend
+from starlette.authentication import AuthenticationBackend
+
+from src.association.settings import PASTAPORTO_SECRET
+from src.webhooks.auth_backend import PretixAuthBackend
+
+
+class WrapperAuthBackend(AuthenticationBackend):
+    def __init__(self):
+        self.pretix_auth_backend = PretixAuthBackend()
+        self.pastaporto_backend = PastaportoAuthBackend(PASTAPORTO_SECRET)
+
+    async def authenticate(self, request):
```

Comment:
doesn't seem to be a better way to handle this :( we need different auth backend based on the path, two middleware override each other